tests/test_clients.py

```diff
@@ -52,7 +52,7 @@ def test_get_observations(self):
 
 
 class TestClient:
-    def setup_method(self, mtd):
```

Comment:
I was about to block the PR over this change until I read an update on the pytest docs: https://docs.pytest.org/en/latest/xunit_setup.html#method-and-function-level-setup-teardown . I was totally unaware of this.

Did you do this because of that change, or ... ?

Reply:
Yeah